src/PluginsSystem/Tools/Microsoft.Performance.Toolkit.Plugins.Cli/Program.cs

```diff
@@ -51,6 +51,10 @@ private static PluginsCli CreatPluginsCli()
                 {
                     WriteIndented = true,
                     PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+                    Converters =
+                    {
+                        new PluginVersionJsonConverter(),
```

Comment:
do we need SemanticVersionJsonConverter here too?

Also, should we just have a common place to initialize all these serializers for both areas?

Reply:
No, PluginManifest doesn't have a property of SemanticVersion. 

We already have SerializationUtils which handles the initializations of these serializers. I don't want to also move the default options for PluginManifest to Plugins.Core because PluginManifest is only used in the Cli. Or do you have any suggestions?

Reply:
I just don't want to accidentally add a SemanticVersion property in PluginManifest down the line and accidentally break things again because we assume the converters are the same.

I think as long as we have round-trip tests for both PluginManifest and PluginMetadata/PluginIdentity de/serialization it should be OK. Those tests would catch those scenarios.